pegasus/sites.v3/code.org/views/privacy.md

```diff
@@ -0,0 +1,221 @@
+Date of Last Revision: April 4, 2016
+
+## About Code.org
+Code.org® is a non-profit dedicated to expanding participation in computer science by making it available in more schools, and increasing participation by women and underrepresented students of color.
+
+Please read this Privacy Policy (“**Privacy Notice**”) carefully before accessing or using the Website.
+
+## Code.org Privacy Principles
+
+Our Privacy Notice is fueled by our commitment to the following Privacy Principles:
+
+
+1. We’re deeply committed to creating a safe and secure learning environment for our students and teachers. We take the protection of this information seriously.
+2. We do not require you to provide any Personal Information in order to try our tutorials, all of which are accessible without a user account. 
+3. The only reason we collect any data from students or teachers is to better succeed at our mission of providing a great computer science education for every student who wants it.  
+4. We do not sell your Personal Information or exploit it for financial gain; we do not sell ads. We are a charitable 501c3 nonprofit and our revenue comes from donations. We established ourselves as a nonprofit so our mission and your trust will not be in conflict with a for-profit motive.
+5. Any student academic data provided by school district partners to 3rd party evaluators for the purpose of evaluating our courses in meeting our mission will be anonymous / de-identified (per standard industry practice). 
```

Comment:
You're right, should be third.  But it seems like the updated policy already says "third" instead of 3rd.